packages/simplescreenrecorder/PKGBUILD

```diff
@@ -0,0 +1,42 @@
+# Maintainer: OSAMC <https://github.com/osam-cologne/archlinux-proaudio>
+# Contributor: this.ven <misgi@ven.uber.space>
+
+_name=ssr
+pkgname=simplescreenrecorder
+pkgver=0.4.4
+pkgrel=1
+pkgdesc='A feature-rich screen recorder that supports X11 and OpenGL'
+arch=(x86_64 aarch64)
+url='https://www.maartenbaert.be/simplescreenrecorder'
+license=(GPL3)
+groups=(pro-audio)
+depends=(alsa-lib desktop-file-utils ffmpeg glu gtk-update-icon-cache jack
+         libgl libpulse libx11 libxext libxfixes libxi libxinerama qt5-base
+         qt5-x11extras)
+makedepends=(git cmake qt5-tools)
+optdepends=('lib32-simplescreenrecorder: OpenGL recording of 32-bit applications
```

Comment:
We shouldn't depend on AUR packages, even optionally, in packages in our repo. Users should be able to use our repo without using the AUR, IMHO.

Reply:
And what if we build that package, too?

Reply:
As discussed in the Matrix chat, that would require enabling the `multilib` repo in our CI.

Let's wait for the input from @cbix on that.

Reply:
I wouldn't add any packages to our main proaudio repo that depend repos other than core/extra/community. If there is some need for it, we may add a separate `proaudio-multilib` repo. Also, recording OpenGL seems to still be an experimental feature and I'd say it's not worth the effort just to support this super tiny edge case :)

Reply:
I see. I put a comment before the `optdepends` line and provide the link to this conversation for transparency on this decision.

Reply:
OK, that doesn't seem to work very well in the CI. I just remove the line.